tsuruclient/apps.py

```diff
@@ -1,16 +1,20 @@
 from tsuruclient.base import Manager as Base
+try:
+    from urllib.parse import urlencode
+except ImportError:
+    from urllib import urlencode
 
 
 class Manager(Base):
     """
     Manage App resources.
     """
 
-    def list(self):
+    def list(self, **kwargs):
         """
-        Get a list of all apps.
+        Get a list of apps by query string.
         """
-        return self.request("get", "/apps")
+        return self.request("get", "/apps?{}".format(urlencode(kwargs)))
```

Comment:
I think you are good to go with: `return self.request("get", "/apps", params=kwargs)`